src/app/template-library/template-library.component.scss

```diff
@@ -0,0 +1,85 @@
+.category-card-container {
+  height: 220px;
+  width: 100%;
+  border-radius: 7px;
+  position: relative
+}
+
+.category-card-container-small {
+  height: 120px;
+  width: 100%;
+  border-radius: 7px;
+  position: relative
+}
+
+.category-card-background {
+  height: 100%;
+  width: 100%;
+  border-radius: 7px;
+  position: absolute;
+  background-image:url('https://images.unsplash.com/photo-1581089778245-3ce67677f718?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+.category-card-overlay {
+  height: 100%;
+  width: 100%;
+  background: rgba(0, 64, 229, 0.6);
+  border-radius: 7px;
+  position: absolute;
+}
+
+.category-card-title-container {
+  height: 100%;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.category-heading {
+  margin-top: 50px;
+  color: #727272;
+}
+
+@media (min-width:1300px) {
+  .template-library-container {
+    width: 80%;
+    padding-top: 80px;
+    margin: auto;
+  }
+
+  .category-card-title {
+    color: white;
+    font-weight: bold;
+    font-size: 30px;
+  }
+
+  .category-card-title-small {
+    color: white;
+    font-weight: bold;
+    font-size: 22px;
+  }
+}
+
+.templates-grid-container {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.menu-header {
+  color: #727274;
```

Comment:
we have font styling defined under `/theme/pfas/_typograph.scss`. Try to avoid customising font related styling inside each component.
Same for other places that customise font stylings